### ISSUE TITLE
Kite: not running in jupyterlab 

### ISSUE THREAD

**Issue by ragyibrahimRNA** [kiteco/issue-tracker#339]
I have completed the below steps. 

I have a kite-pro subscription with the desktop app installed on macOS 10.15.6
I'm running jupyter-lab 2.2.2 
below is my pip list: 

`%pip list`

Package                     Version
--------------------------- -----------
alembic                     1.4.2
altair                      4.1.0
appnope                     0.1.0
argon2-cffi                 20.1.0
astropy                     4.0.1.post1
attrs                       19.3.0
autopep8                    1.5.3
azure-core                  1.7.0
azure-storage-blob          12.3.2
backcall                    0.2.0
bleach                      3.1.5
blis                        0.4.1
boto                        2.49.0
boto3                       1.14.34
botocore                    1.17.34
cachetools                  4.1.1
catalogue                   1.0.0
catboost                    0.23.2
certifi                     2020.6.20
cffi                        1.14.0
chardet                     3.0.4
click                       7.1.2
clickclick                  1.2.2
cloudpickle                 1.5.0
cmdstanpy                   0.4.0
colorlover                  0.3.0
combo                       0.1.1
confuse                     1.3.0
connexion                   2.7.0
convertdate                 2.2.1
cryptography                2.9.2
cufflinks                   0.17.3
cycler                      0.10.0
cymem                       2.0.3
Cython                      0.29.21
databricks-cli              0.11.0
datefinder                  0.7.1
DateTime                    4.3
decorator                   4.4.2
defusedxml                  0.6.0
descartes                   1.1.0
docker                      4.2.2
docutils                    0.15.2
entrypoints                 0.3
ephem                       3.7.7.1
fbprophet                   0.6
flake8                      3.8.3
Flask                       1.1.2
funcy                       1.14
future                      0.18.2
gensim                      3.8.3
gitdb                       4.0.5
GitPython                   3.1.7
google-api-core             1.17.0
google-api-python-client    1.10.0
google-auth                 1.18.0
google-auth-httplib2        0.0.3
google-auth-oauthlib        0.4.1
google-cloud-bigquery       1.25.0
google-cloud-core           1.3.0
google-cloud-logging        1.15.0
google-cloud-scheduler      1.3.0
google-cloud-secret-manager 1.0.0
google-resumable-media      0.5.1
googleapis-common-protos    1.52.0
gorilla                     0.3.0
graphviz                    0.14.1
grpc-google-iam-v1          0.12.3
grpcio                      1.27.2
gunicorn                    19.3.0
holidays                    0.10.2
htmlmin                     0.1.12
httplib2                    0.18.1
idna                        2.10
ImageHash                   4.1.0
imbalanced-learn            0.7.0
importlib-metadata          1.7.0
inflection                  0.5.0
iniconfig                   1.0.1
ipykernel                   5.3.2
ipython                     7.16.1
ipython-genutils            0.2.0
ipywidgets                  7.5.1
isodate                     0.6.0
itsdangerous                1.1.0
jedi                        0.17.1
Jinja2                      2.11.2
jmespath                    0.10.0
joblib                      0.16.0
jsonschema                  3.2.0
jupyter-client              6.1.6
jupyter-core                4.6.3
**jupyter-kite                1.0.0**
kiwisolver                  1.2.0
kmodes                      0.10.2
korean-lunar-calendar       0.2.1
lightgbm                    2.3.1
llvmlite                    0.33.0
LunarCalendar               0.0.9
Mako                        1.1.3
MarkupSafe                  1.1.1
matplotlib                  3.3.0
mccabe                      0.6.1
missingno                   0.4.2
mistune                     0.8.4
mizani                      0.7.1
mlflow                      1.10.0
mlxtend                     0.17.3
more-itertools              8.4.0
msrest                      0.6.18
murmurhash                  1.0.2
nbconvert                   5.6.1
nbformat                    5.0.7
networkx                    2.4
nltk                        3.5
notebook                    6.1.1
numba                       0.50.1
numexpr                     2.7.1
numpy                       1.19.0
oauth2client                4.1.3
oauthlib                    3.1.0
openapi-spec-validator      0.2.8
packaging                   20.4
palettable                  3.3.0
pandas                      1.1.0
pandas-gbq                  0.13.2
pandas-profiling            2.8.0
pandocfilters               1.4.2
parso                       0.7.0
patsy                       0.5.1
pexpect                     4.8.0
phik                        0.10.0
pickleshare                 0.7.5
Pillow                      7.2.0
pip                         20.1.1
plac                        1.1.3
plotly                      4.9.0
plotnine                    0.7.0
pluggy                      0.13.1
preshed                     3.0.2
prometheus-client           0.8.0
prometheus-flask-exporter   0.15.4
prompt-toolkit              3.0.5
protobuf                    3.12.2
ptyprocess                  0.6.0
py                          1.9.0
pyasn1                      0.4.8
pyasn1-modules              0.2.8
pycaret                     2.0
pycodestyle                 2.6.0
pycparser                   2.20
pycryptodome                3.9.8
pydata-google-auth          1.1.0
pyflakes                    2.2.0
Pygments                    2.6.1
pyLDAvis                    2.1.2
PyMeeus                     0.3.7
PyMySQL                     0.9.3
pyod                        0.8.1
pyparsing                   2.4.7
pyrsistent                  0.16.0
pystan                      2.19.1.1
pytest                      6.0.1
python-dateutil             2.8.1
python-dotenv               0.14.0
python-editor               1.0.4
pytz                        2020.1
PyWavelets                  1.1.1
PyYAML                      5.3.1
pyzmq                       19.0.1
querystring-parser          1.2.4
regex                       2020.7.14
requests                    2.24.0
requests-oauthlib           1.3.0
retrying                    1.3.3
rsa                         4.6
s3transfer                  0.3.3
scikit-learn                0.23.1
scipy                       1.5.2
seaborn                     0.10.1
Send2Trash                  1.5.0
setuptools                  49.2.0
setuptools-git              1.2
six                         1.15.0
smart-open                  2.1.0
smmap                       3.0.4
spacy                       2.3.2
SQLAlchemy                  1.3.13
sqlparse                    0.3.1
srsly                       1.0.2
statsmodels                 0.11.1
suod                        0.0.4
swagger-ui-bundle           0.0.8
tabulate                    0.8.7
tangled-up-in-unicode       0.0.6
terminado                   0.8.3
testpath                    0.4.4
textblob                    0.15.3
thinc                       7.4.1
threadpoolctl               2.1.0
toml                        0.10.1
toolz                       0.10.0
tornado                     6.0.4
tqdm                        4.48.2
traitlets                   4.3.3
ua-parser                   0.10.0
umap-learn                  0.4.6
uritemplate                 3.0.1
urllib3                     1.25.9
user-agents                 2.1
vega-datasets               0.8.0
visions                     0.4.4
wasabi                      0.7.1
wcwidth                     0.2.5
webencodings                0.5.1
websocket-client            0.57.0
Werkzeug                    1.0.1
wheel                       0.34.2
widgetsnbextension          3.5.1
wordcloud                   1.7.0
xgboost                     1.1.1
yellowbrick                 1.1
zipp                        3.1.0
zope.interface              5.1.0

`!jupyter labextension list`
JupyterLab v2.2.2
Known labextensions:
   app dir: /usr/local/Cellar/jupyterlab/2.2.2/libexec/share/jupyter/lab
        @kiteco/jupyterlab-kite v1.0.0  enabled  OK

my jupyterlab-kite is not working. See screenshot below. I get no errors on installation

![Screen Shot 2020-08-09 at 11 22 33 am](https://user-images.githubusercontent.com/52903382/89722916-c9f94d00-da32-11ea-9b03-8d67772d955c.png)

![Screen Shot 2020-08-09 at 11 27 52 am](https://user-images.githubusercontent.com/52903382/89722957-8a7f3080-da33-11ea-82ec-8700458d8347.png)

**Comment by its-dhung** [comment]
What are the logs that are printed to your browser's JavaScript console as you type code into the notebook?

**Comment by its-dhung** [comment]
Possibly related to #318

**Comment by ragyibrahimRNA** [comment]
There's nothing in the console...does not mention Kite at all 

![Screen Shot 2020-08-18 at 12 10 34 pm](https://user-images.githubusercontent.com/52903382/90462441-391a2400-e14c-11ea-9fc4-f44184e8483d.png)

**Comment by edzkite** [comment]
Closing as duplicate of https://github.com/kiteco/issue-tracker/issues/380

**Comment by edzkite** [comment]
@ragyibrahimRNA Would you mind sharing the output of `jupyter serverextension list`? Also, the output in your terminal when you run `jupyter lab`?
